### ISSUE TITLE
Images en mode portrait ?

### ISSUE THREAD

**Issue by Thevenin-Jonathan** [racacax/XML-TV-Fr#60]
Salut ! Déjà un gros merci pour ce soft qui est très utile !

Maintenant ma question, est-il possible d'avoir les images en mode portrait ? J'utilise PLEX et ca rend vraiment moche les images en mode paysage zoomé... 

Merci !

**Comment by racacax** [comment]
Salut,
Merci du soutien, ça fait plaisir.

Je comprends le problème. Certains providers proposent différentes images
et des fois en mode portrait. Ajouter une option pour privilégier le mode
portrait lorsque c'est possible est tout à fait envisageable.

A noter que ce serait uniquement sur la version soft et non pour le XML
déjà fourni sur xmltvfr.fr à moins qu'il y ait une option dans Plex qui
permette de choisir automatiquement l'image avec les meilleures dimensions.
Je sais qu'on peut indiquer plusieurs images par programme ainsi que les
dimensions.

Le mar. 12 mars 2024, 14:23, Thevenin Jonathan ***@***.***> a
écrit :

**Comment by Thevenin-Jonathan** [comment]
@racacax Non, dans PLEX, on ne choisit rien pour le LIVE TV, soit on utilise leurs EPG, qui est très bien et dans le bon sens, mais aucune chaine Canal+, soit on passe par un xml perso, ce que je fais avec ton soft mais tout les images sont en 4x3 et non en 2x3...

Sur les EPG Orange, FREE et SFR de Plex, toutes les images sont en 2x3, donc je me disais que par chance, vous les aviez au même endroit et que c'était une option à ajouter. En tout cas ca serait TRES utile car c'est vraiment très moche :(

En tout cas je te remercie pour ta rapidité de réponse <3

**Comment by racacax** [comment]
Je viens de regarder et cette option est possible pour certains providers. Orange ne fournit pas d'image 2x3 (du moins dans l'endpoint que j'utilise), mais SFR par exemple le propose (ex : http://static-cdn.tv.sfr.net/data/img/pl/7/0/4/6714407.jpg et http://static-cdn.tv.sfr.net/data/img/pl/9/0/4/6714409.jpg pour le même programme)

Un paramètre permettant de prioriser les images serait envisageable. Je garde cette issue sous le coude et devrait l'ajouter prochainement.

**Comment by Thevenin-Jonathan** [comment]
Salut ! Ah super bonne nouvelle çà ! Je pense que ca pourrait servir à d'autre que moi ! Je te remercie pour le temps que tu as passer pour ma requête et te souhaite bon courage pour l'implémenter :)
